packages/app/src/specs/flaky-badge/FlakySpecSummary.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div
+    class="border-t-4px min-w-200px w-full max-w-400px grid p-4 gap-4 grid-cols-1 justify-items-center"
+    :class="accentClass"
+    data-cy="flaky-spec-summary"
+  >
+    <SpecNameDisplay
+      :spec-file-name="specName"
+      :spec-file-extension="specExtension"
+    />
+    <div class="flex flex-row w-full text-size-14px justify-center items-center">
+      <component :is="severityIcon" />
+      <span
+        class="font-medium ml-2"
+        :class="textClass"
+      >{{ severityLabel }}</span>
+      <span
+        class="ml-4"
+        data-cy="flaky-rate"
+      >{{ t('specPage.flaky.flakyRate', [flakyRate]) }}</span>
+    </div>
+
+    <div class="w-full grid text-gray-700 text-size-14px gap-2 grid-cols-2 justify-items-center">
+      <span data-cy="flaky-runs">{{ t('specPage.flaky.flakyRuns', [totalFlakyRuns, totalRuns]) }}</span>
+      <span data-cy="last-flaky">{{ t('specPage.flaky.lastFlaky', [lastFlaky]) }}</span>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+
+import { computed, FunctionalComponent, ref, shallowRef, watch } from 'vue'
+import { useI18n } from '@cy/i18n'
+import SpecNameDisplay from '../SpecNameDisplay.vue'
+import LowRateIcon from '~icons/cy/rate-low_x16'
+import MediumRateIcon from '~icons/cy/rate-medium_x16'
+import HighRateIcon from '~icons/cy/rate-high_x16'
+import ceil from 'lodash/ceil'
+
+const { t } = useI18n()
+
+const props = defineProps<{
+  specName: string
+  specExtension: string
+  severity: 'low' | 'medium' | 'high'
+  totalFlakyRuns: number
+  totalRuns: number
+  lastFlaky: number
+  dashboardUrl: string
+}>()
+
+const accentClass = ref<string | null>(null)
```

Comment:
Instead of explicit `null` it looks like you can do

```js
const accentClass = ref<string>()
```

Which gives the type of `string | undefined`. Bit more concise - unless there's a compelling reason to use `null`, this seems a bit better.